### DISCUSSION TITLE
Homebrew not working on M1 Mac mini

### DISCUSSION THREAD

**Discussion by chrisjniles** [Homebrew/discussions#715] [Getting started] [answered]
I installed Homebrew on my M1 Mac mini running the public release of Big Sur (no beta for me), but when I try to use it I get an error message: `zsh: command not found: brew`. I tried to run `brew doctor` and `brew config` and get the same error message.

**Comment by carlocab** [comment, marked as answer]
Did you follow the instructions shown at the end of the install?

Here's what it would show me:
```
- Add Homebrew to your PATH in /Users/carlocab/.zprofile:
    echo 'eval $(/opt/homebrew/bin/brew shellenv)' >> /Users/carlocab/.zprofile
    eval $(/opt/homebrew/bin/brew shellenv)
```

The message will have been different on your machine.

**Reply by chrisjniles** [reply]
Nope, I'm a dolt and completely skipped over that
